### ISSUE TITLE
Last received message is not immediately visible to the user when opening a conversation

### ISSUE THREAD

**Issue by hanspeide** [intercom/intercom-android#242]
## Version info
- Intercom for Android version: 3.0.3
- Android OS version: 6.0.0
- Android Device: Nexus 5X (Genymotion)
## Expected behavior

When opening a conversation using `Intercom.client().displayMessenger()`, the list of messages should be scrolled all the way to the bottom. By doing this the last received message will be immediately visible to the user. 
## Actual behavior

When opening a conversation the list is scrolled to the top, so that the first message sent/received by the customer is shown.

A side effect of this is that the app bar is jumping.
## Steps to reproduce
1. User must have a number of sent/received messages in a conversation
2. Open the conversation using `Intercom.client().displayMessenger()`
3. Observer that the list is scrolled to the top, and the latest received message is not shown until the user has scrolled to the bottom of the list

**Comment by ronocod** [comment]
Video of bug in action: https://www.youtube.com/watch?v=3a6JzMff24c (thanks for this 😄 )

**Comment by DaleCantwell** [comment]
I have re-evaluated this issue. I believe this is following our ruleset. Until the conversation has a reply from a team member we stay at the top to show the profile.
Please let us know if you are seeing the stay at top behaviour in a conversation that has replies from admins and we can re open this.

**Comment by igorvilela28** [comment]
I'm having problems with that too. I want to ALWAYS start with the bottom scroll, showing the last received message. Even when we got responses, the messages sometimes are starting at the top.

It would be awersome if we could have a settings that decide how we want the chat conversation to behaviour.

I recorded a video to show:

[Link to video](https://drive.google.com/file/d/0B48-65XMhMA1elFzcnVtWWI1Z1E/view?usp=sharing)
